docs/README.macOS.md

```diff
@@ -9,12 +9,12 @@ This guide has been tested with macOS 10.13.4()17E199 High Sierra and Xcode 9.3(
 3. **[Get the source code](#3-get-the-source-code)**
 4. **[Configure and build tools and dependencies](#4-configure-and-build-tools-and-dependencies)**
 5. **[Build binary add-ons](#5-build-binary-add-ons)**
-6. **[Build Kodi](#6-build-kodi)**  
-  6.1. **[Build with Xcode](#61-build-with-xcode)**  
-  6.2. **[Build with xcodebuild](#62-build-with-xcodebuild)**  
+6. **[Build Kodi](#6-build-kodi)**
```

Comment:
Removing the double spaces is wrong. Its the markdown for a newline. Id revert any of these changes

Reply:
Ah, ok, did not realise. Will fix